README.md

```diff
@@ -50,6 +50,10 @@ To disable the plugin
 let g:loaded_nvimgdb = 1
 ```
 
+`:GdbStart` and `:GdbStartLLDB` use `find` and the cmake file API to try to
+tab-complete the command with the executable for the current file. To disable
+this set `g:use_find_executables` or `g:use_cmake_to_find_executables` to `0`.
```

Comment:
May I suggest a bit more plugin-specific names for the variables? For instance, `g:gdb_use_find_executables` etc.